pkg/paragliderpb/paraglider.proto

```diff
@@ -69,6 +69,10 @@ message ResourceDescriptionString {
     string description = 2;
 }
 
+message ResourceString {
```

Comment:
I don't think this should be here, right?

Reply:
Fixed